main.go

```diff
@@ -58,5 +58,15 @@ func main() {
 	l.Printf("Inventory Service, version %s starting up",
 		CreateVersionString())
 
+	db, err := NewDataStoreMongo(config.Config.GetString(SettingDb))
+	if err != nil {
+		l.Fatal("failed to connect to db")
+	}
+
+	err = db.Migrate(Version, nil)
```

Comment:
@maciejmrowiec no, we don't have to do it every release. we only do it when an actual migration is  needed, until then everything stays at the initial version (hence the jira comment about gaps in schema versions - they're ok). 

so there's no way to forget anything, it'll be a lot of targeted manual work.